sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -115,14 +116,8 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 3e762e00bcf13d158fb58e8e8c2eabcfc8934b4e
-  # Merge: c805a71a a2a053de
-  # Author: Omar Ahmed <omar.ahmed@codeplay.com>
-  # Date:   Wed Jul 31 12:26:34 2024 +0100
-  #     Merge pull request #1884 from callumfare/callum/fix_printtrace
-  #     Enable PrintTrace when SYCL UR tracing is enabled
-  set(UNIFIED_RUNTIME_TAG 3e762e00bcf13d158fb58e8e8c2eabcfc8934b4e)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/zhaomaosu/unified-runtime.git")
```

Comment:
patch was merged, please update the commit

Reply:
Thanks. Updated.

Reply:
Hi @pbalcer, looks like the UR tag is ahead of my fix, so I revert tag updating part. May I get your approval for this change? Thanks.